lisa/nic.py

```diff
@@ -437,7 +435,7 @@ def _load_nics(self) -> None:
                     self.nics[nic].pci_slot = pci_device.slot
                 break
 
-    def is_mana_present(self) -> bool:
+    def is_mana_device_present(self) -> bool:
```

Comment:
We can change the original `_is_mana_device_discovered` to the new name, and remove this public method. It just call private method, no other logic.